addons/l10n_id_efaktur/tests/test_l10n_id_efaktur.py

```diff
@@ -82,8 +82,9 @@ def test_efaktur_csv_output_decimal_place(self):
             _csv_row(LT_HEAD_LIST, ','),
             _csv_row(OF_HEAD_LIST, ','),
         )
-        line_4 = '"FK","01","0","0000000000002","5","2019","1/5/2019","12345","l10ntest","","40040","4004","0","","0","44044","0","0","INV/2019/00002 12345","0"\n'
-        line_5 = '"OF","","","100","400.0","40040","0","40040","4004","0","0"\n'
+        line_4 = '"FK","01","0","0000000000002","5","2019","1/5/2019","12345","l10ntest","","40040","4004","0","","0","44044.0","0.0","0","INV/2019/00002 12345","0"\n'
+        # value should be 100.1 but float_round returns the 100.10000000000001
+        line_5 = '"OF","","","100.10000000000001","400.0","40040.0","0.0","40040.0","4004.0","0","0"\n'
```

Comment:
This test should be failing with the float_repr :thinking:

Reply:
right, I'm confirming the if it's functionally correct with ELON first, then I'll push to fix the test case!